charts/datadoc/values.schema.json

```diff
@@ -93,79 +49,40 @@
+    "features": {
+      "description": "Feature toggles",
+      "type": "object",
+      "properties": {}
+    },
+    "tjeneste": {
+      "description": "Velg versjon av tjenesten",
       "type": "object",
       "properties": {
-        "requests": {
-          "description": "The amount of CPU requested",
+        "image": {
+          "description": "image docker",
           "type": "object",
           "properties": {
-            "cpu": {
-              "description": "The amount of CPU guaranteed",
-              "title": "CPU",
+            "pullPolicy": {
               "type": "string",
-              "default": "100m",
-              "render": "slider",
-              "sliderMin": 50,
-              "sliderMax": 8000,
-              "sliderStep": 50,
-              "sliderUnit": "m",
+              "description": "option when pulling the docker image",
+              "default": "Always",
               "x-onyxia": {
-                "overwriteDefaultWith": "region.resources.cpuRequest",
-                "useRegionSliderConfig": "cpu"
-              }
+                "hidden": true
+              },
+              "enum": ["IfNotPresent", "Always", "Never"]
             },
-            "memory": {
-              "description": "The amount of memory guaranteed",
-              "title": "Memory",
+            "version": {
+              "description": "Datadoc-versjon",
               "type": "string",
-              "default": "1Gi",
-              "render": "slider",
-              "sliderMin": 1,
-              "sliderMax": 600,
-              "sliderStep": 1,
-              "sliderUnit": "Gi",
-              "x-onyxia": {
-                "overwriteDefaultWith": "region.resources.memoryRequest",
-                "useRegionSliderConfig": "memory"
-              }
+              "default": "v0",
+              "listEnum": ["v0", "master"],
+              "render": "list",
+              "pattern": "^[a-z0-9-_./]+(:[a-z0-9-_.]+)?$"
             }
           }
         }
       }
     },
-    "diskplass": {
-      "description": "Configuration for persistence",
-      "type": "object",
-      "properties": {
-        "enabled": {
-          "type": "boolean",
-          "description": "Create a persistent volume",
-          "default": true
-        },
-        "size": {
-          "type": "string",
-          "title": "Persistent volume size",
-          "description": "Size of the persistent volume",
-          "default": "10Gi",
-          "form": true,
-          "render": "slider",
-          "sliderMin": 1,
-          "sliderMax": 100,
-          "sliderStep": 1,
-          "sliderUnit": "Gi",
-          "x-onyxia": {
-            "overwriteDefaultWith": "region.resources.disk",
-            "useRegionSliderConfig": "disk"
-          },
-          "hidden": {
-            "value": false,
-            "path": "diskplass/enabled"
-          }
-        }
-      }
-    },
```

Comment:
I'm not sure that it works just to remove these blocks entirely. I think we need to provide a value to the Onyxia API so we need to have something here.

It might work just setting hidden to true and enabled default to false